drivers/mysql/mysql.go

```diff
@@ -487,9 +544,17 @@ func (m *Mysql) Info() (*schema.Driver, error) {
 		name = "mariadb"
 	}
 
+	dct := dict.New()
+	dct.Merge(map[string]string{
+		"Subroutines": "Stored procedures and functions",
```

Comment:
```suggestion
		"Functions": "Stored procedures and functions",
```

I've given it a lot of thought.
How about I proceed with the following naming?

Because, the word "Function" is considered to cover the word "Procedure", and the new word "Subroutine" is not used. I tried not to use it.

Reply:
I would be happy to have the other "Subroutine" changed as well!

Reply:
fixed in [569f959](https://github.com/k1LoW/tbls/pull/341/commits/569f959e2155bd0747c27959c4b7bcf4587daa4e) and [55d046a](https://github.com/k1LoW/tbls/pull/341/commits/55d046a52e4ec53c8dc8230c822bd51e0a14828a). Two commits as I have some issues with my WSL (my primary OS is windows, so using WSL to build the product on linux) and there are some issues with it. Thanks!